[homework #4] - CSS by Nicolás Roberto

nicolasroberto opened the pull request:
First commit of css homework. I still need to do the dark mode.

solvd-mmelnyk commented:
@nicolasroberto please do rebase for this branch since I merged your previous homework - I should not see files related to previous homework

nicolasroberto commented:
> @nicolasroberto please do rebase for this branch since I merged your previous homework - I should not see files related to previous homework

My bad, i was working from gitbash and i didn't see i had my previous homework in this folder too. Now that i added the git extension to my terminal i can see the folders in the selected branch. I removed the #02 folder from the #04 homework.

solvd-mmelnyk commented:
> > @nicolasroberto please do rebase for this branch since I merged your previous homework - I should not see files related to previous homework
> 
> My bad, i was working from gitbash and i didn't see i had my previous homework in this folder too. Now that i added the git extension to my terminal i can see the folders in the selected branch. I removed the #2 folder from the #4 homework.

yes, I see that you deleted folders
but you should do next:
1. pull latest main branch
2. move to your branch
3. do git rebase for you branch from main

if it is too hard you can ask mentor to help

nicolasroberto commented:
> yes, I see that you deleted folders but you should do next:
> 
> 1. pull latest main branch
> 2. move to your branch
> 3. do git rebase for you branch from main
> 
> if it is too hard you can ask mentor to help

Ok i did it. If it's wrong i'm going to ask mentors.

nicolasroberto commented:
Do i need to commit again after rebase?

solvd-mmelnyk commented:
> > yes, I see that you deleted folders but you should do next:
> > 
> > 1. pull latest main branch
> > 2. move to your branch
> > 3. do git rebase for you branch from main
> > 
> > if it is too hard you can ask mentor to help
> 
> Ok i did it. If it's wrong i'm going to ask mentors.

yes, please ask in chat and somebody will help you

nicolasroberto commented:
> yes, please ask in chat and somebody will help you

I got the help by the mentors. It should be working now. But we didn't do the rebase, so the folder 02-html it's still appearing in my nr-04-css branch after i reopened vscode.

solvd-mmelnyk commented on homeworks/nicolas-roberto/04-css/index.html:
```diff
@@ -0,0 +1,185 @@
+        <p>Software Enginieer</p>
+        <ul class="socialmedia--icons">
+          <li>
+            <a href=""><img src="./icon/github.svg" alt="" /></a>
```
Please add alt texts for all img tags

solvd-mmelnyk commented on homeworks/nicolas-roberto/04-css/index.html:
```diff
@@ -0,0 +1,185 @@
+    <main>
+      <!-- facts-information start -->
+      <div class="facts">
+        <ul class="profile--facts">
```
Please use BEM approach for all classes: https://en.bem.info/methodology/quick-start/

solvd-mmelnyk commented:
@nicolasroberto good work!
